docs/scripting-doc/index.d.ts

```diff
@@ -5327,6 +5380,23 @@ declare class Dialog extends Qt.QWidget {
    */
   addFilePicker(labelText?: string): FileEdit;
 
+  /**
+   * Add a {@link QButtonGroup} widget which allows you to add multiple radio
+   * buttons, where only one radio button can be selected at a time.
+   *
+   * Each radio button is a {@link QAbstractButton}. You can force selection of
+   * a radio button by setting checked = true, and also disable
+   *
+   * @param labelText - If the labelText is non-empty, a label widget will be added to the left of
+   *  the widget.
+   * @param values - Each entry in the array is the text that will appear on the radio button.
+   * @param labelToolTip - Optional tooltip for the label to the left of the radio button group.
+   * @param buttonToolTips - Optional list of tooltips, where each entry is a tooltip that corresponds to the radio button in the list of values with the same index.
```

Comment:
To make the online docs a little less verbose, I've removed the elements related to params. So this will not be shown and needs to be moved into the description without using the `@param` directive.

Reply:
I think I took care of both of these, let me know